pkg/networkservice/common/monitor/server_test.go

```diff
@@ -0,0 +1,268 @@
+package monitor
```

Comment:
Please move tests to `monitor_test` package

Reply:
I can't because test use internals to check.